src/helpers/formatDate.js

```diff
@@ -0,0 +1,12 @@
+/** Formateer naar een voor mensen leesbare datum**/
+
+const formatDate = (isoDate) =>{
```

Comment:
in uitwerkingen is dit een normale functie ipv arrowfunctie. maar als goed is is beide prima